spec/presenters/tree_builder_policy_profile_spec.rb

```diff
@@ -0,0 +1,9 @@
+describe TreeBuilderPolicyProfile do
+  context '#tree_init_options' do
+    it "is explicitly not lazy" do
+      tree = TreeBuilderPolicyProfile.new(:policy_profile_tree, :policy_profile, {}, true)
+      options = tree.instance_variable_get(:@options)
+      expect(options[:lazy]).to eq(false)
```

Comment:
Unfortunately no. I mean, I can test that calling the method doesn't fail .. but I can't test that expanding the node in the browser does not fail on the ruby side, we'd need intergration tests for that.